### ISSUE TITLE
Make some sort of basic handout (for judges)?

### ISSUE THREAD

**Issue by WasabiFan** [kyranstar/Hunt-The-Wumpus#55]
I think we should print out a list of the big bullets that we think make our game great and amazing so that we can give it to the judges for reference. From what I have heard of the format of this, there will be a lot of judges judging a lot of teams, so having a quick reminder for them (w/ screenshots) would probably keep us memorable for them. What do you guys think we should put on something like this?

My thoughts are:
- Game judging
  - Kept with a consistent visual theme ("old-style" arcade)
  - Included animations where appropriate to make playing the game less jarring
  - Kept a modern-looking style for all of the graphical elements
  - Gave a full UI that seems comprehensive
- Code judging
  - Didn't cut corners -- fully implemented things with the proper extension methods, class structures, etc
  - Incorporated more complex programming concepts, such as reflection, C# attributes and XAML generation
  - Wrote all the render code ourselves
  - Wrote more complex algorithms where appropriate
    - Pathfinding
    - Random map generation
  - Used a state machine to control game flow
  - No hard-coded data (all loaded from files)
  - Cross-platform with no Win32 API calls

What else? I really think that having a list like this could give us a good shot at being at the top -- from what I have seen, our game is already very good compared to most of them.

**Comment by WasabiFan** [comment]
@kyranstar Any ideas here? I'd really like to get this done so we have it tomorrow, and would like others' input before I make it

**Comment by nikbyk** [comment]
@WasabiFan You said that the game can be displayed on a WinPhone. If that works, it may be worth showing.

**Comment by WasabiFan** [comment]
Yeah; I'm working on fixing the little issues so that it's demo-able on the phone. Hopeefully it'll be on my device and ready to run tomorrow.

Do you have any other ideas for textual bullets for a handout-type thing?

**Comment by nikbyk** [comment]
no, you seemed to cover everything pretty well. My one concern is that the Wumpus is way too active, should we turn off the active wumpus? Especially if we don't see it in game, it may be hard to demo

**Comment by WasabiFan** [comment]
Yeah... I think that's a good plan. Would you be willing to fix that? You should be able to find the proper line to comment out in the Wumpus object. -- it should be pretty straightforward.

I'm currently working on more finalization tasks -- is there anything else that you think needs to be done tonight?

**Comment by nikbyk** [comment]
It may be a good idea to render room numbers with low opacity on the rooms, otherwise most of the secrets are useless.

**Comment by WasabiFan** [comment]
Yeah... that's (another) good point. I think I will wait until tomorrow and finalize that there, so that I can make sure all of the critical things are done, but that's high on the list. Honestly, I'm feeling pretty sad that I didn't get a chance to add the buttons and theme switcher... I might try that tomorrow as well.

P.S. I just successfully deployed the app to my phone and it runs, so we should be able to demo. As long as you don't need to move, of course.

**Comment by nikbyk** [comment]
Great! I'll work on the active wumpus

**Comment by WasabiFan** [comment]
OK; thanks!

**Comment by nikbyk** [comment]
Done, but our cam commands don't work, and the game hangs every now and then when I start

**Comment by WasabiFan** [comment]
I just fixed the cam command, and the freezing is the issue with pathfinding from #53. So, it's not your fault -- you can push your changes and it shouldn't cause any problems.

**Comment by nikbyk** [comment]
My thing didn't work for active wumpus, please assist

**Comment by nikbyk** [comment]
cam command still does not work :/

**Comment by WasabiFan** [comment]
> cam command still does not work :/

Even after pulling? It seems to work for me

> My thing didn't work for active wumpus, please assist

You should be able to comment out line 80 in `Wumpus.cs`:
```
CurrentBehavior.Move();
```

I haven't tested it, but I think that's what you want.

**Comment by nikbyk** [comment]
I think I tried that, cam doesn't work and I pulled

**Comment by nikbyk** [comment]
Yea, line 80 doesn't help

**Comment by WasabiFan** [comment]
@kyranstar We're confused... When you get a chance, can you please take a look at this and disable the active Wumpus? We don't understand how it works...

**Comment by WasabiFan** [comment]
> cam doesn't work and I pulled

This works for me:
```
> cam 0.1
```

**Comment by nikbyk** [comment]
tried, just flashes once and that's it...

**Comment by nikbyk** [comment]
can you pull and try? maybe I broke something

**Comment by WasabiFan** [comment]
That _really_ sounds like you haven't pulled -- I just synced and it works for me, and I can confirm that I don't have any local changes that I haven't pushed.

**Comment by nikbyk** [comment]
that's really weird because I just pulled

**Comment by WasabiFan** [comment]
Hmmmm... I don't know then. Try `Project -> Rebuild Solution`?

**Comment by nikbyk** [comment]
I'm cleaning now

**Comment by nikbyk** [comment]
clean didn't help;
rebuilding

**Comment by nikbyk** [comment]
no luck :/

**Comment by anak4569** [comment]
Yo guys, is the wumpus supposed to appear when you get close? It makes shooting him really easy.

**Comment by nikbyk** [comment]
he's always visible

**Comment by nikbyk** [comment]
shouldn't be tho

**Comment by anak4569** [comment]
yea, shouldn't we make it so he isn't always visible?

**Comment by WasabiFan** [comment]
That's something that I plan to sort out tomorrow, unless someone else gets to it first. Although if we don't get to doing that we can claim that it's for demo purposes...

**Comment by anak4569** [comment]
Has anyone already entered the building?
